### DISCUSSION TITLE
Max_line_width doesn't work for me

### DISCUSSION THREAD

**Discussion by petqo** [openai/whisper#1548] [Q&A] [answered]
Hello,

I use whisperAI in google colaboratory, works fine and fast but for some reason --max_line_width doesn't work, does anyone has this same problem? 

here's the command I use

!whisper untitled.mp3 --language cs --model medium --word_timestamps True --max_line_width 16

**Comment by ryanheise** [comment]
Can you copy/paste part of the generated `srt` file to illustrate the problem?

**Reply by petqo** [reply]
1
00:00:01,500 --> 00:00:09,820
Hrubé
znamení,
které se
snaží
odradit
každého, kdo
by chtěl
proniknout
hlouběji do
jeskyně.
Držte se
dál,
nebezpečí
uvnitř,
zákaz
vstupu.

2
00:00:10,580 --> 00:00:12,780
Dobře, jdu
tam, co
nejhoršího
se může
stát.

3
00:00:13,580 --> 00:00:24,280
Ty pičo! Co
ti jebe!
Hoši! Hoši!
Hoši! Hoši!
To si děláte
prdel!

**Reply by ryanheise** [reply]
For comparison, can you paste the output with `--max_line_width 32`?

**Reply by petqo** [reply]
1
00:00:01,500 --> 00:00:09,820
Hrubé znamení, které se snaží
odradit každého, kdo by chtěl
proniknout hlouběji do jeskyně.
Držte se dál, nebezpečí uvnitř,
zákaz vstupu.

2
00:00:10,580 --> 00:00:12,780
Dobře, jdu tam, co nejhoršího se
může stát.

3
00:00:13,580 --> 00:00:24,280
Ty pičo! Co ti jebe! Hoši! Hoši!
Hoši! Hoši! To si děláte prdel!

**Reply by petqo** [reply]
Oh so it's not like the lines matches the timestamps and makes "shorter" subtitles... My "goal" is to make shorter lines with timestamps, cause I want to use that for tiktok videos, where you give only 1-4 words to the screen

**Reply by ryanheise** [reply, marked as answer]
You can use `--max_line_width` in conjunction with `--max_line_count`. The former decides when to start a new line, and the latter decides when to start a new subtitle. Note that the latter overrides Whisper's native intuition on when to start a new subtitle which often (but not always) aligns with phrase and sentence boundaries, so by overriding that you will just get it starting a new subtitle whenever the length limit has been reached. There is one exception where it will try to start a new subtitle earlier than the length limit if there has been a pause of a few seconds.

**Reply by petqo** [reply]
thank you so much!!

**Reply by wenchaoliu-93** [reply]
> You can use `--max_line_width` in conjunction with `--max_line_count`. The former decides when to start a new line, and the latter decides when to start a new subtitle. Note that the latter overrides Whisper's native intuition on when to start a new subtitle which often (but not always) aligns with phrase and sentence boundaries, so by overriding that you will just get it starting a new subtitle whenever the length limit has been reached. There is one exception where it will try to start a new subtitle earlier than the length limit if there has been a pause of a few seconds.

It doesn't work for me.
Command: `time whisper He\ Is\ Risen.mp4 --model tiny  --output_format srt --max_line_count 1 --max_line_width 42 --word_timestamps True --device cpu`

Sample Output: `[00:02.000 --> 00:14.360]  This has been a remarkable session, and behalf of all who participated thus far in Word or music.
[00:16.580 --> 00:22.860]  As the President of the Church, I've chosen simply to say to you on this moment.
[00:24.700 --> 00:30.780]  And just two words, known as the two most important words in the English language,
`

**Reply by wenchaoliu-93** [reply]
Sorry. It looks like the whisper output on the terminal can be different from the actual output file.
